src/applications/personalization/view-representative/components/ViewRepresentativeDetails.jsx

```diff
@@ -1,11 +1,39 @@
 import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
 
 export const ViewRepresentativeDetails = props => (
   <>
-    <h2 className="vads-u-margin-top--1p5">
-      Your selected representative for VA claims
-    </h2>
-    <div className="vads-u-background-color--gray-lightest vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-padding-top--1 vads-u-padding-bottom--2 vads-u-padding-x--2 vads-u-font-size--md vads-u-font-family--sans">
+    <h2 className="vads-u-font-size--h3 vads-u-margin-top--6">On this page</h2>
+    <ul className="usa-unstyled-list vads-u-margin-top--2">
+      <li>
+        <a href="#selectedRepresentative">
+          <i className="fas fa-arrow-down va-c-font-size--xs vads-u-margin-top--1 vads-u-margin-right--1" />
```

Comment:
For these icons, it may be good to add `aria-hidden` or `role=image` + `aria-label=<some text to indicate navigation to ancho>` since the icon itself doesn't hold much meaning for screen readers.

Reply:
updated this. It looks like just adding `aria-hidden` is enough, adding an `aria-label` will just double up what is read by screenreaders.